src/AspNetCoreModuleV2/AspNetCore/HandlerResolver.cpp

```diff
@@ -56,14 +59,35 @@ HandlerResolver::LoadRequestHandlerAssembly(IHttpApplication &pApplication, STRU
 
             RETURN_IF_FAILED(location.Copy(options->GetExeLocation()));
 
-            if (FAILED_LOG(hr = FindNativeAssemblyFromHostfxr(options.get(), pstrHandlerDllName, &struFileName)))
+            RETURN_IF_FAILED(LoggingHelpers::CreateLoggingProvider(
+                pConfiguration->QueryStdoutLogEnabled(),
+                !GetConsoleWindow(),
+                pConfiguration->QueryStdoutLogFile()->QueryStr(),
+                pApplication.GetApplicationPhysicalPath(),
+                &outputManager));
+
+            outputManager->Start();
+
+            hr = FindNativeAssemblyFromHostfxr(options.get(), pstrHandlerDllName, &struFileName);
+            outputManager->Stop();
+
+            if (FAILED(hr) && m_hHostFxrDll != NULL)
             {
+                STRA content;
+                STRU struStdMsg;
+
+                outputManager->GetStdOutContent(&content);
+                if (content.QueryCCH() > 0)
+                {
+                    struStdMsg.CopyA(content.QueryStr());
```

Comment:
Note for the future, we should profile the call to CopyA as I was seeing it take ~500 ms in the debugger.